.github/workflows/test.yml

```diff
@@ -1,6 +1,6 @@
 name: Test
 
-on: [push]
+on: [push, pull_request]
```

Comment:
That will cause multiple test runs for pro from internal branches. Take a look at how we fix this issue in e.g. Vega-lite.

Reply:
Thanks, fixed!